### PR TITLE
feat(animal-group): documentation for animal group

### PR DESCRIPTION
**Pull request by ryanjosephtorrado**
----

Notes

----

## PR Checklist
- [x] Documentation has been checked by at least one team member
- [x] Links work to navigate documentation
- [ ] Images stored in `_images`
- [x] Spell checked

> If some of the above are not completed why?
> **REASON HERE**

### CODE REVIEW

File: docs/resource-types/livestock/animal-group.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    - api
+    - data
+  post_tag:
+    - api
+---
+
+# Resource Type: Animal Group
+
+- [URLs]()
+- [Response Structure]()
```
Comment (danielR9) [inline docs/resource-types/livestock/animal-group.md:17]: I think these need actual links to the sections below

File: docs/resource-types/livestock/animal-group.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+| **Birth Period**        | The range of birth dates of this Animal Group                                                                                                                         | String               |
+| **Count**               | The count or number of animals in this inventory classification                                                                                                       | Decimal              |
+| **Lactation Status**    | The lactation status of animals. (Valid values: Dry, Lead, Fresh, Early, Lactating)                                                                                   | Enumeration          |
+| **Name**                | Human-readable name for this inventory grouping                                                                                                                       | String               |
+| **Primary Breed**       | ICAR breed code for this Animal Group. For example:<br/>`{ "scheme": "ICAR2", "id": "AN" }`<br/>For more details, see also Reference Data Breeds - PureFarming DevHub | Indentifier          |
```
Comment (danielR9) [inline docs/resource-types/livestock/animal-group.md:99]: *Identifier
Also link to the common identifier object

File: docs/resource-types/livestock/animal-group.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  "reproductiveStatus": "String",
+  "sex": "String",
+  "species": "String"
+}
+```
+
+| Response Item           | Description                                                                                                                                                           | Data Type            |
+| ----------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------- |
+| **Birth Period**        | The range of birth dates of this Animal Group                                                                                                                         | String               |
+| **Count**               | The count or number of animals in this inventory classification                                                                                                       | Decimal              |
```
Comment (danielR9) [inline docs/resource-types/livestock/animal-group.md:96]: Not sure what we are doing elsewhere but I'm not so sure on `Decimal`, I think I would prefer `Number`? (or integer, but its not an int in the schema I don't think

File: docs/resource-types/livestock/animal-group.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+{
+  "birthPeriod": "String",
+  "count": decimal,
+  "lactationStatus": "String",
+  "name": "String",
+  "primaryBreed": {...},
+  "productionPurposes": [...],
+  "reproductiveStatus": "String",
+  "sex": "String",
+  "species": "String"
```
Comment (danielR9) [inline docs/resource-types/livestock/animal-group.md:89]:
```suggestion
  "birthPeriod": "string",
  "count": 1,
  "lactationStatus": "string",
  "name": "string",
  "primaryBreed": {...},
  "productionPurposes": [...],
  "reproductiveStatus": "string",
  "sex": "string",
  "species": "string"
```

File: docs/resource-types/livestock/animal-group.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+}
+```
+
+| Response Item      | Description                                                                                    | Data Type      |
+| ------------------ | ---------------------------------------------------------------------------------------------- | -------------- |
+| **Classification** | The set of shared characteristics that defines the group of animals                            | Classification |
+| **Id**             | The Pure Farming Id of this Animal Group                                                       | UUID           |
+| **Self**           | A link to this specific Animal Group                                                           | URI            |
+| **Location**       | An identifier for the location of the Animal Group                                             | Identifier     |
+| **Meta**           | Meta data for the resource                                                                     | Metadata       |
```
Comment (danielR9) [inline docs/resource-types/livestock/animal-group.md:70]: The types Classification, Identifier and Metadata should all be links to where the object is defined

File: docs/resource-types/livestock/animal-group.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+A call to the animal group endpoints returns the following fields.
+
+```json
+{
+  "classification": {...},
+  "id": "string",
+  "self": "string",
+  "location": {...},
+  "meta": {...},
+  "resourceType": "livestock/animal-group"
```
Comment (danielR9) [inline docs/resource-types/livestock/animal-group.md:60]:
```suggestion
  "resourceType": "/livestock/animal-group"
```

File: docs/resource-types/livestock/animal-group.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+post_excerpt: Animal - Group
+taxonomy:
+  category:
+    - api
+    - data
+  post_tag:
+    - api
+---
+
+# Resource Type: Animal Group
```
Comment (danielR9) [inline docs/resource-types/livestock/animal-group.md:14]: I'm not sure whether we are going for "Animal Group" vs "Animal Groups" in this title - do whatever is consistent with most of the other documentation pages

File: docs/resource-types/livestock/index.md [hunk 8]
```diff
@@ -1,22 +1,21 @@
...
+    - root
+    - livestock
 ---
 
 # Resource Types: Livestock
 
 Below are the currently provided resource types for Livestock.
 
 - [Animals](/resource-types/livestock/animals.md)
+- [Animal Group](/resource-types/livestock/animal-group.md)
```
Comment (danielR9) [inline docs/resource-types/livestock/index.md:20]: Make the plural here consistent with whatever we title the main page as

**Comment by ryanjosephtorrado** [comment]
Hey @danielR9 , I've updated the documentation. Please check and review. Thank you

File: docs/resource-types/livestock/animal-group.md [hunk 9]
```diff
@@ -57,18 +57,18 @@ A call to the animal group endpoints returns the following fields.
...
+| ------------------ | ---------------------------------------------------------------------------------------------- | ------------------------------------------------------- |
+| **Classification** | The set of shared characteristics that defines the group of animals                            | [Classification](#classification)                       |
+| **Id**             | The Pure Farming Id of this Animal Group                                                       | UUID                                                    |
+| **Self**           | A link to this specific Animal Group                                                           | URI                                                     |
+| **Location**       | An identifier for the location of the Animal Group                                             | [Identifier](/docs/resource-types/common.md#identifier) |
+| **Meta**           | Meta data for the resource                                                                     | [Metadata](/docs/resource-types/common.md#metadata)     |
```
Comment (danielR9) [inline docs/resource-types/livestock/animal-group.md:70]:
```suggestion
| **Meta**           | Meta data for the resource                                                                     | [Metadata](/resource-types/common.md#metadata)     |
```

**Review by danielR9 — CHANGES_REQUESTED** [review]

**Review by danielR9 — APPROVED** [review]

**Pull request merged**